File: go/models/Group.go
package models

import "github.com/go-pg/pg"

type TblGroup struct {
	tableName struct{} `sql:"tbl_group,alias:t" pg:",discard_unknown_columns"`

	ID          string      `sql:"id,pk"`
	Avatar      *string     `sql:"avatar"`
	Background  *string     `sql:"background"`
	CreateTime  pg.NullTime `sql:"create_time"`
	CreatorID   string      `sql:"creator_id,notnull"`
	DeleteTime  pg.NullTime `sql:"delete_time"`
	Description *string     `sql:"description"`
	MemberNum   *int        `sql:"member_num"`
	Name        *string     `sql:"name"`
}

File: src/main/java/com/shumsg/model/entity/Group.java
package com.shumsg.model.entity;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class Group {
    private String id;

    private String name;

    private String description;

    private String avatar;

    private String background;

    private String creatorId;

    private Integer memberNum;

    private LocalDateTime createTime;

    private LocalDateTime deleteTime;

}